modules/rules_ios/3.0.0/MODULE.bazel

```diff
@@ -0,0 +1,125 @@
+"""
+Defines all the external repositories and dependencies for rules_ios.
+"""
+
+# Defines the rules_ios bzlmod module.
+# Version is updated during release to the registry.
+module(
+    name = "rules_ios",
+    version = "3.0.0",
```

Comment:
This version should match the declared module version (`3.0.0`). Are you using the Publish to BCR app? It should automatically generate a patch that does this for you.

@meteorcloudy It looks like we are missing a check that enforces module version and metadata version to be identical.

Reply:
+1 to adding a check for version, we planned to use the app but when we tagged this version it wasn't configured correctly so it never ran. Is it possible to trigger it manually?